dbft_test.go

```diff
@@ -228,7 +281,7 @@ func newTestState(myIndex int, count int) *testState {
 	s := &testState{
 		myIndex: myIndex,
 		count:   count,
-		ch:      make(chan Payload, 1),
+		ch:      make(chan Payload, 4),
```

Comment:
hm it's changed for all tests to 4. And interesting why is that so =)

Reply:
I initially approached this the wrong way. Testing routine should never hang on channel send so I have replaced this channel with slice and removed magic constants.